src/components/common/Footer.css

```diff
@@ -0,0 +1,133 @@
+.section__padding {
+	padding: 4rem 4rem;
+}
```

Comment:
missing `box-sizing: border-box`,  on smaller devices, without it the footer is wider than the screen width
<img width="400" alt="Screenshot 2023-05-12 at 23 43 15" src="https://github.com/fairwear/frontend-fairwear/assets/74358398/6f16bd97-d003-4d60-b640-4d13beb52c69">